pkg/reconciler/route/traffic/rollout.go

```diff
@@ -21,7 +21,9 @@ limitations under the License.
 
 package traffic
 
-import "knative.dev/networking/pkg/apis/networking"
+import (
+	"knative.dev/networking/pkg/apis/networking"
+)
```

Comment:
Who are you, and what have you done with @vagababov ?

Reply:
I am expecting more imports :)